sdk/oracle/Azure.ResourceManager.OracleDatabase/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Specify the expected release date by replacing the `Unreleased`, the format is `yyyy-mm-dd`.